pwa-backend/src/main/java/io/growingabit/app/model/StudentConfirmationEmail.java

```diff
@@ -39,6 +41,15 @@ public Long getTsExpiration() {
     return this.tsExpiration;
   }
 
+  public String getOriginHost() {
+    return this.originHost;
+  }
+
+  public void setOriginHost(String originHost) {
+    Preconditions.checkArgument(originHost != null);
```

Comment:
Controlla sempre anche stringa vuota, se vuoto non è accettato (come in questo caso)